Java/core/src/main/java/com/amazon/randomcutforest/tree/HyperTree.java

```diff
@@ -123,7 +123,8 @@ private Cut getCut(IBoundingBoxView bb, Random ring, Function<IBoundingBoxView,
     }
 
     @Override
-    public void deletePoint(double[] point, long sequenceIndex) {
+    public double[] deletePoint(double[] point, long sequenceIndex) {
+        return point;
         // () -> /dev/null
```

Comment:
I think we can remove this comment.

Reply:
Done. Removed comment from the corresponding add as well.